templates/puppet.conf

```diff
@@ -1,14 +1,10 @@
+# {{ ansible_managed }}
 [main]
-    logdir = /var/log/puppet
-    rundir = /var/run/puppet
-    ssldir = $vardir/ssl
 
 [agent]
-    classfile = $vardir/classes.txt
-    localconfig = $vardir/localconfig
     server = {{ ansible_fqdn }}
```

Comment:
The puppetmaster doesn't really run an agent. Looking closer at puppet.conf reference, these should probably go to [main] or [master]

Reply:
Hmm.. I'll test. I think it could be nice to be able to run "puppet agent -t" on the puppetmaster too rather than the masterless bit in this role or am I wrong?

Reply:
True! I've mostly run the puppet apply thing on the puppetmasters, but OTOH can't think of a reason why we couldn't support running the agent.

Maybe if we were doing the "periodic agent poll + immediate provisioning" thing, then I wouldn't want the puppetmaster agent to immediately apply what has just been merged to production. But in any case that's not a thing right now.

Reply:
Ran some testing and without the [agent] bits then indeed running "puppet agent -t" does not work on the puppetmaster -- I would like to keep it :)

Reply:
Yup, resolved